main.go

```diff
@@ -125,6 +126,14 @@ func main() {
 
 	ctx := signals.SetupSignalHandler()
 
+	signingRequestor, err := certificate.StartSigningRequestor(broker.SyncerConfig{
+		LocalRestConfig: restConfig,
+		LocalClient:     dynClient,
+		RestMapper:      restMapper,
+		Scheme:          scheme.Scheme,
+	}, ctx.Done())
+	logger.FatalOnError(err, "Error creating SigningRequestor")
```

Comment:
This should be started by the gateway when it starts leading prior to starting the cable engine so it should be internal to the gateway.